docs/workforce-management-framework/topics/con-datasync-cloud-module.adoc

```diff
@@ -0,0 +1,42 @@
+= {WFM-RC-NameLong} Datasync Module
```

Comment:
should the heading be {WFM-RC-NameLong} Datasync Cloud given that the name of the adoc is "con-datasync-cloud-module.adoc"?

Reply:
renamed the file